### DISCUSSION TITLE
Custom s3 bucket for specific collection

### DISCUSSION THREAD

**Discussion by manamelacedric** [pocketbase/pocketbase#5902] [Q&A]
Hi all,

I would like to use a different s3 bucket for one of my collections. Is this possible?
I am willing to do the go implementation if someone can point me in the right direction.

Thanks

**Comment by ganigeorgiev** [comment]
No, it is not possible. The S3 storage configuration can be only one for the entire application and there are no plans to allow configuration for this.

**Reply by manamelacedric** [reply]
Alright, thanks.

Ill use the filesystem.NewS3 to copy the file to s3 in an OnModelAfterCreate hook

**Reply by ganigeorgiev** [reply]
I don't recommend doing this as it could cause unintended side-effects on record update/delete.

It is fine if PocketBase doesn't fit your use case requirements and you may find it easier to use something else (or build it from scratch).
